modules/core/src/main/java/org/apache/ignite/client/ClientCache.java

```diff
@@ -205,13 +364,29 @@
      */
     public void removeAll(Set<? extends K> keys) throws ClientException;
 
+    /**
+     * Removes entries for the specified keys.
+     * <p>
+     * The order in which the individual entries are removed is undefined.
+     *
+     * @param keys The keys to remove.
+     */
+    public IgniteClientFuture<Void> removeAllAsync(Set<? extends K> keys) throws ClientException;
+
     /**
      * Removes all of the mappings from this cache.
      * <p>
      * The order that the individual entries are removed is undefined.
      */
     public void removeAll() throws ClientException;
 
+    /**
+     * Removes all of the mappings from this cache.
+     * <p>
+     * The order that the individual entries are removed is undefined.
```

Comment:
Maybe it needs to mention if the operation is atomic

Reply:
Fixed